composer.json

```diff
@@ -22,11 +22,19 @@
     },
     "require-dev": {
         "composer/installers": "^1.9",
-        "johnpbloch/wordpress": "^5.4",
+        "johnpbloch/wordpress": "^6.1",
         "wp-graphql/wp-graphql": "^1.1.8",
         "squizlabs/php_codesniffer": "^3.5",
         "automattic/vipwpcs": "^2.3",
-        "wp-coding-standards/wpcs": "^2.3"
+        "wp-coding-standards/wpcs": "^2.3",
+        "wp-phpunit/wp-phpunit": "^6.3",
+        "yoast/phpunit-polyfills": "^2.0",
+        "phpunit/phpunit": "^9.6",
+        "codeception/module-asserts": "*",
+        "codeception/util-universalframework": "*",
+        "codeception/module-rest": "*",
+        "lucatume/wp-browser": "^3.1",
+        "php-coveralls/php-coveralls": "2.4.3"
```

Comment:
Note the fixed version here. Not sure if we want to do something different, but this is the newest version that I could get to cooperate with the other dependencies & PHP >= 8.1.

Tested in my own CI run:

![image](https://github.com/wp-graphql/wp-graphql-testcase/assets/4269287/fd7b3d83-3316-4811-8ef1-268a797e7373)

From:

PHP 8.1: https://github.com/missionmike/wp-graphql-testcase/actions/runs/8098665304/job/22132597313?pr=1
PHP 8.2: https://github.com/missionmike/wp-graphql-testcase/actions/runs/8098665304/job/22132597637?pr=1
PHP 8.3: https://github.com/missionmike/wp-graphql-testcase/actions/runs/8098665304/job/22132597954?pr=1